FMS/Pages/Facilities/Details.cshtml

```diff
@@ -133,7 +199,7 @@
             <tbody>
                 @foreach (var r in Model.FacilityDetail.RetentionRecords)
                 {
-                    <tr class="@(r.Id == Model.HighlightRecord ? "table-warning" : "") @(!r.Active ? "text-muted" : "") ">
+                    <tclass ="@(r.Id == Model.HighlightRecord ? "table-warning" : "") @(!r.Active ? "text-muted" : "") ">
```

Comment:
Typo in element name. "<tclass =" should be "<tr class="

Reply:
Thanks, I wondered what happened with this. I did a merge of your .NET 8 changes and rebased and many typos occurred in this file. I missed this because it works, but it gave warnings. Thanks for picking this up, I was scratching my head about how to correct it.